ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -344,8 +344,12 @@ jobs:
   serial: true
   plan:
   - get: geode
+{%- if repository.upstream_fork != "apache" or repository.branch == "develop" %}
+    passed: [PublishArtifacts]
+{% else %}
     passed:
-    {{ all_gating_jobs() | indent(4) }}
+    {{ all_gating_jobs() | indent(6) }}
```

Comment:
Not sure it will matter since it's he gating enumeration, but does the change from `indent(4)` to `indent(6)` matter?

Reply:
The other block is identically indented and does work well. Additionally, the `passing` block is taking a list provided by this macro, so i think it will be well.